### ISSUE TITLE
TODOs

### ISSUE THREAD

**Issue by pannous** [pannous/tensorflow-speech-recognition#29]
- [ ] split test set(s) #28
- [ ] make input->chars converge well  (input->class works well already)
- [ ] sliding window
- [ ] merge WarpCTC or alternative
- [ ] peer2peer training!
